### DISCUSSION TITLE
<Hi, can anyone test my AI image creator for any bugs or issues >

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65725] [Programming Help]
### Select Topic Area

Product Feedback

### Body

my website link 🔗 https://electrosion.web.app
My website is new and it may contain some bugs or issues i posted this for testing purpose

**Comment by NovoaDev** [comment]
Hi, first of all I am not a web developer nor a UX/UI expert, I will give you my humble opinion as a user.

if I try to use the "Start using" button it breaks. :)
![image](https://github.com/community/community/assets/21259250/223ce182-dc14-48a8-b97d-7d5db3dbf126)

Besides I would like to put some pading to the page(In my ignorance).
![image](https://github.com/community/community/assets/21259250/576a7e96-b6c8-4cb2-a2bd-f13e262094bb)

I would remove annoying pop-ups:
![image](https://github.com/community/community/assets/21259250/d259f060-b7bf-4bec-8868-5bba36d8cc06)
![image](https://github.com/community/community/assets/21259250/e14a6697-adff-40a0-b1c0-65a6de06abc6)

Otherwise I see it well, I did some tests and the result is acceptable.

Well done.

**Reply by ghost** [reply]
hi,thanks for your valuable feedback i have fixed 'serverless function has crashed error' and also removed alert box and will soon try to replace 'join discord' popup

**Comment by of1r** [comment]
I don't know if there are any other pages but the main menu doesn't work.

**Comment by Binary-Ninja-007** [comment]
Good.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @yasir-io ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
